extensions/sql-migration/src/api/utils.ts

```diff
@@ -312,19 +311,19 @@ export async function getAzureTenantsDropdownValues(tenants: Tenant[]): Promise<
 	if (tenantsValues.length === 0) {
 		tenantsValues = [
 			{
-				displayName: constants.ACCOUNT_SELECTION_PAGE_NO_LINKED_ACCOUNTS_ERROR,	//
+				displayName: constants.ACCOUNT_SELECTION_PAGE_NO_LINKED_ACCOUNTS_ERROR,
 				name: ''
 			}
 		];
 	}
 	return tenantsValues;
 }
 
-export async function getAzureSubscriptions(account?: azdata.Account): Promise<azureResource.AzureResourceSubscription[]> {
+export async function getAzureSubscriptions(account?: Account): Promise<azureResource.AzureResourceSubscription[]> {
 	let subscriptions: azureResource.AzureResourceSubscription[] = [];
 	try {
 		if (account) {
-			subscriptions = !account.isStale ? await getSubscriptions(account) : [];
+			subscriptions = !account.isStale ? await azure.getSubscriptions(account) : [];
```

Comment:
I am not 100% sure, but I think since account.isStale can be undefined you can only rely on isState===false to indicate it is in ready state
![image](https://user-images.githubusercontent.com/61598682/165856819-bbb2e170-cdc7-4f63-9eed-2caf6ae8c98d.png)